### DISCUSSION TITLE
Upgrade from 4.0.0-beta2 to 4.1.0

### DISCUSSION THREAD

**Discussion by lakano** [THM-Health/PILOS#1501] [Q&A] [answered]
Hi !

I would like to upgrade from the beta2 to the new release 4.1.0

So, I'm checking all warnings / informations between theses versions, and on the 4.0.0-rc3, you said :
« The issue where the last meeting of a room is not shown on the home page is only fixed for upgrades from PILOS v2/v3. To manually fix this issue please run the following command in the database: UPDATE `rooms` as rooms SET `meeting_id`= (SELECT id FROM `meetings` as meetings WHERE meetings.room_id = rooms.id ORDER BY `meetings`.`created_at` DESC LIMIT 1); »

This manual fix is to do for all versions before the 4.0.0-rc3 (including my beta2) or is it only for those upgrading from 4.0.0-rc2 ?

Thanks!

**Comment by samuelwei** [comment, marked as answer]
This issue exists if the "artisan db:upgrade" command was run before 4.0.0-rc3 - so yes you have to run the fix command
